### PR TITLE
Add recommended distribution for jitter and add Stefan Holmer as author.

### PR DESCRIPTION
**Pull request by stefanholmer**

### CODE REVIEW

File: draft-ietf-rmcat-eval-criteria.xml [hunk 1]
```diff
@@ -69,6 +69,10 @@
 
         <author fullname="Stefan Holmer" initials="S." surname="Holmer">
           <organization>Google</organization>
+        </author>
+
+        <author fullname="Stefan Holmer" initials="S." surname="Holmer">
```
Comment (vr000m) [inline draft-ietf-rmcat-eval-criteria.xml]: Could you remove the previous author block.

**Comment by stefanholmer** [comment]
Done

**Comment by vr000m** [comment]
I was hoping we could reduce the introduction on the jitter section?

**Comment by stefanholmer** [comment]
Yeah, that might be possible, but I came to realize that it might be good
to have some text which describes how to apply the jitter (z(n)) in a
reasonable way, which that text does in a very correct way. We may be able
to reduce it a bit, but maybe it makes it more difficult to understand for
someone not used to this stuff.

**Comment by vr000m** [comment]
Alright. Makes sense.

**Pull request merged**